src/pages/VenuePage/VenuePage.tsx

```diff
@@ -181,6 +200,7 @@ export const VenuePage: React.FC = () => {
   }
 
   if (!venue || !venueId) {
+    // if !venueId is true loading page might display indefinitely, another message or action may be appropriate
```

Comment:
have you had that behavior? looks like `venueId` is taken from the url. And if there is no `venueId` it gets redirected to https://sparklespaces.com/ (at least this is how it works for me)

Reply:
@sunny-viktoryia This is kind of a guard for a situation that is unlikely if not impossible in our current setup, but could potentially become an issue if the routes/implementation we use to get the `venueId`/etc changes.

There are some patterns I used in my recent `src/pages/Account/Questions.tsx` refactor that solve this need in a better way. I linked them on slack to @goran-peoski-work, it may be worth just adding that here to solve this potential edge case once and for all?

Reply:
Maybe next PR, a smaller change, faster to review. I'm hesitant of adding more things in this one., that might be OOS.

I put a comment that would be able to pinpoint to go back and replace with proper message/action, once/if product/tech decide course.